pkg/cloudprovider/providers/gce/gce_clusters.go

```diff
@@ -41,24 +45,30 @@ func (gce *GCECloud) ListClusters(ctx context.Context) ([]string, error) {
 }
 
 func (gce *GCECloud) GetManagedClusters(ctx context.Context) ([]*container.Cluster, error) {
-	managedClusters := []*container.Cluster{}
-	for _, zone := range gce.managedZones {
-		clusters, err := gce.getClustersInZone(zone)
-		if err != nil {
-			return nil, err
-		}
-		managedClusters = append(managedClusters, clusters...)
+	var location string
+	if len(gce.managedZones) > 1 {
+		// Multiple managed zones means this is a regional cluster
```

Comment:
I don't know where this is called from, but we support both multi-zonal clusters and regional clusters. So managing multiple zones does not imply that the cluster is regional, which may or may not make a difference depending on what the purpose of this code is.